POS/Operation/02_SelectOffer.md

```diff
@@ -1,17 +1,86 @@
 [!!Manage offers for POS](../Integration/07_ManageOffers.md)
 [!!User interface Cashpoint](../UserInterface/01a_Cashpoint.md)
 
-# Select an offer
+# Record the purchase
 
-The *Venduo POS* module offers several ways to select an offer or to search for it.
-You may select an offer:
-- by Quickselect, see [Select by QuickSelect](#select-by-quickselect)
-- by name, see [Select by name](#select-by-name)
-- by number, see [Select by number](#select-by-number)
+The *Venduo POS* module offers several ways to select an offer and to search for it. Additionally, you can prepare further actions for editing the bill list, to ensure the protection of minors, to create returns, and to add product options.  
```

Comment:
Record the purchase --> several ways to select an offer and to search for it -> Erster Satz passt nicht mehr zum Titel. Diese Info unter "Select an offer" weiter unten und hier Intro-Info anpassen?

Reply:
done